componentsPage.html

```diff
@@ -207,7 +207,45 @@ <h2>Icon Buttons</h2>
            </section>
 
            <section class="section" id="image-container">
-            image Will Come Here...
+               <h1>Images</h1>
+               <h2>Responsive Images</h2>
+               <p>It is very important to use responsive images in our website.
+
+                To create responsive image you need to add <span>image-responsive</span> class to the element</p>
+
+                <div class="image-responsive">
+                    <img src="https://images.pexels.com/photos/2353413/pexels-photo-2353413.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500" alt="img">
```

Comment:
In the alt tag img is written which is redundant, because screen reader alredy pronounce img tag as image.
So you should add something else there instead of image.